[0m17:25:40.743578 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E48510FFD0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E48554D9D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E485206B50>]}


============================== 17:25:40.751664 | 54a0ee49-2dcc-4ed4-add0-9093e0f8c2b1 ==============================
[0m17:25:40.751664 [info ] [MainThread]: Running with dbt=1.5.2
[0m17:25:40.753814 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\Tal\\.dbt', 'debug': 'False', 'fail_fast': 'False', 'log_path': 'logs', 'warn_error': 'None', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'target_path': 'None', 'send_anonymous_usage_stats': 'True'}
[0m17:25:40.761950 [debug] [MainThread]: Starter project path: C:\Users\Tal\AppData\Local\Programs\Python\Python311\Lib\site-packages\dbt\include\starter_project
[0m17:25:53.096694 [info ] [MainThread]: Profile sakila_dbt written to C:\Users\Tal\.dbt\profiles.yml using target's sample configuration. Once updated, you'll be able to start developing with dbt.
[0m17:25:53.097702 [info ] [MainThread]: 
Your new dbt project "sakila_dbt" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m17:25:53.100020 [debug] [MainThread]: Command `dbt init` succeeded at 17:25:53.100020 after 12.38 seconds
[0m17:25:53.101024 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E4833BF650>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E485477710>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000001E4851E08D0>]}
[0m17:25:53.102026 [debug] [MainThread]: Flushing usage events
[0m14:59:03.443048 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002E2B0A92ED0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002E2B6D3EF90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002E2B6842050>]}


============================== 14:59:03.451810 | 13363365-5144-494f-a2d3-6e87bbba93df ==============================
[0m14:59:03.451810 [info ] [MainThread]: Running with dbt=1.5.2
[0m14:59:03.453970 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\Tal\\.dbt', 'version_check': 'True', 'warn_error': 'None', 'log_path': 'logs', 'debug': 'False', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'introspect': 'True', 'log_format': 'default', 'target_path': 'None', 'send_anonymous_usage_stats': 'True'}
[0m14:59:03.455290 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\sakila_dbt_course\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m14:59:03.457754 [debug] [MainThread]: Command `dbt run` failed at 14:59:03.457754 after 0.05 seconds
[0m14:59:03.459184 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002E2B719DAD0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002E2B6842050>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002E2B6D3E410>]}
[0m14:59:03.460602 [debug] [MainThread]: Flushing usage events
